rustls-platform-verifier/src/verification/windows.rs

```diff
@@ -34,16 +34,18 @@ use std::{
     ptr::{self, NonNull},
     sync::Arc,
 };
+use windows_sys::Win32::Security::Cryptography::CERT_TRUST_IS_PARTIAL_CHAIN;
 use windows_sys::Win32::{
```

Comment:
Nit: Merge this import into the other block for consistency?

Reply:
Sure, I've done it